Requests/ChangeMembersRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using PyrusApiClient;
+using System.Collections.Generic;
+
+namespace Pyrus.ApiClient.Requests
+{
+    public class ChangeMembersRequest
```

Comment:
nit: stylistically stands out from other data contracts and there is not enough enter
Let's do it like this
```c#
	public List<Person> AddMembers { get; set; }

	public List<Person> ChangeMembers { get; set; }

	public list<Person> BanMembers { get; set; }
```